### PR TITLE
Scc 3961

### PR DESCRIPTION
**Pull request by danamansana**
- Remove check for `resp` to be truthy in case of 400, since HTTPError content is apparently falsey

### CODE REVIEW

File: lib/poller.py [hunk 1]
```diff
@@ -120,9 +120,9 @@ def send_notifications(self, entries):
             if resp and resp.status_code == 200:
                 # Mark hold as processed:
                 self.redis_client.set_hold_processed(entry)
-            elif resp and resp.status_code == 400 and str(resp.content).find('Unable to find patron email') > -1:
-                # If the notification fails because the patron is missing an email,
-                # log the issue but do not retry
+            # If the notification fails because the patron is missing an email,
+            # log the issue but do not retry
+            elif resp.status_code == 400 and str(resp.content).find('Unable to find patron email') > -1:
```
Comment (nonword) [inline lib/poller.py:125]: Is it possible `resp` is `None`?

File: lib/poller.py [hunk 2]
```diff
@@ -120,9 +120,9 @@ def send_notifications(self, entries):
...
                 self.logger.warning('Unexpected response from PatronServices'
                                     + f' notify endpoint for {path} {payload}'
```
Comment (nonword) [inline lib/poller.py:127]: While we're here, could we also cange this from "Unexpected response" to something that indicates it's somewhat expected like "Got no-email response from PatronServices"?

File: lib/poller.py (hunk 1 above)
Comment (danamansana) [reply lib/poller.py:125]: Good point! I will check that `resp is not None`

File: lib/poller.py (hunk 2 above)
Comment (danamansana) [reply lib/poller.py:127]: 👍

**Review by nonword — APPROVED** [review]

**Pull request merged**